content/events/wicked-bendathalon.md

```diff
@@ -0,0 +1,41 @@
+@title 1v1 Wicked Bender
+
+# 1v1 Wicked Bender (aka Glory Wicked-Bender, aka Wicked Bendathalon)
```

Comment:
We should decide on one thing to call it, this is too long for a title

Reply:
OKay. Let's do Wicked Bendathlon. That is what the event commissioners have agreed on.